code/modules/antagonists/brother/brother.dm

```diff
@@ -28,10 +28,17 @@
 	return ..()
 
 /datum/antagonist/brother/proc/equip_brother()
+	var/mob/living/carbon/human/brother = owner.current //it's probably fine if we cast this directly to human, because the procs being called don't care
 	var/obj/item/book/granter/crafting_recipe/weapons/W = new
-	W.on_reading_finished(owner.current)
+	W.on_reading_finished(brother)
 	qdel(W)
 
+	var/obj/item/storage/box/bloodbrother/T = new()
+	if(brother.equip_to_slot(T, ITEM_SLOT_BACKPACK)) //except for here, where it will fail because there's no backpack slot to equip to
+		SEND_SIGNAL(brother.back, COMSIG_TRY_STORAGE_SHOW, brother)// which is fine, because it'll just not do anything and give a notice (probably)
```

Comment:
runtimes should be handled properly because they cause lag and in the worst cases, unpredictable behavior & catastrophic failure
```suggestion
		SEND_SIGNAL(brother.back, COMSIG_TRY_STORAGE_SHOW, brother)
```